### DISCUSSION TITLE
Prevent q-epansion-item content to load before first open

### DISCUSSION THREAD

**Discussion by kirschkern** [quasarframework/quasar#14094] [General - Components / Directives / etc]
Im using a description page with a lot of entries. Each entry is a q-expansion-item.
When opening an entry I see all details including images and videos. However, I only want this content to load, when the entry gets opened as otherwise a lot of data will be loaded at once.

Is there any setting I can use? Or do you have a good idea how to solve this?

My only idea is to handle the show event to set a flag which will then allow the content to be shown.
```
<q-expansion-item @show="shown=true">
  <div v-if="shown">
  ...
  </div>
</q-expansion-item>
```

Any better solution?

**Comment by pdanpdan** [comment]
There is this that can be turned to a PR
https://github.com/pdanpdan/quasar/commit/ce6cb171bee7ad28c43563d72be9c307b11bd8b2

**Reply by kirschkern** [reply]
Looks promising, How can I use it?
